Description: Do not use integer samples when __SOFTFP__ is defined
 On armel __SOFTFP__ is defined. Hence in 1.7.1 SAMPLETYPE changed from float to
 short. Unfortunately, this changes some function signatures and thus breaks the
 ABI.
Author: Sebastian Ramacher <sramacher@debian.org>
Bug-Debian: http://bugs.debian.org/723681
Forwarded: not-needed
Last-Update: 2013-09-19

--- a/include/STTypes.h
+++ b/include/STTypes.h
@@ -89,13 +89,6 @@
     /// runtime performance so recommendation is to keep this off.
     // #define USE_MULTICH_ALWAYS
 
-    #if (defined(__SOFTFP__))
-        // For Android compilation: Force use of Integer samples in case that
-        // compilation uses soft-floating point emulation - soft-fp is way too slow
-        #undef  SOUNDTOUCH_FLOAT_SAMPLES
-        #define SOUNDTOUCH_INTEGER_SAMPLES      1
-    #endif
-
     #if !(SOUNDTOUCH_INTEGER_SAMPLES || SOUNDTOUCH_FLOAT_SAMPLES)
        
         /// Choose either 32bit floating point or 16bit integer sampletype
